conda/plugins/hookspec.py

```diff
@@ -208,3 +209,31 @@ def conda_auth_handlers():
                     auth_handler=EnvironmentHeaderAuth,
                 )
         """
+
+    @_hookspec
+    def conda_post_solves(self) -> Iterable[CondaPostSolve]:
+        """
+        Register post-solve functions in conda.
```

Comment:
```suggestion
        Register post-solve functions in conda that are used in the
        general solver API, after the solver has provided the package
        records to add or remove from the conda environment.
```

Reply:
in conda that are called after the solver has [solved]